bower.json

```diff
@@ -2,10 +2,8 @@
   "name": "angular-google-places-autocomplete",
   "version": "0.2.4",
   "main": [
-      "/src/autocomplete.js",
-      "/src/autocomplete.css",
-      "/dist/autocomplete.min.js",
-      "/dist/autocomplete.min.css"
```

Comment:
According to this specification (https://github.com/bower/bower.json-spec#main), main section should included full version of files such as sources or compiled sources(from CoffeeScript, etc.)